addons/mail/static/src/messaging/component/partner_mention_suggestion/partner_mention_suggestion.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.PartnerMentionSuggestion" owl="1">
+        <t t-if="partner and composer">
+            <t t-if="partner.divider">
+                <div class="dropdown-divider" role="separator"/>
+            </t>
+            <a class="o_PartnerMentionSuggestion dropdown-item" t-att-class="{ 'active': partner.id === composer.active_id }" href="#" role="menuitem" t-att-data-id="partner.id" t-on-click="_onClickMentionItem" t-on-mouseover="_onMouseOverMention">
+                <PartnerImStatusIcon
+                    partnerMentionSuggestionDisplay="true"
+                    partnerLocalId="partner ? partner.localId : undefined"
```

Comment:
I'm not sure to understand here, what is the use case where we want a `PartnerImStatusIcon` but we don't have a partner?
Shouldn't the condition be above, so that we don't render it at all otherwise?
Maybe there is a reason, but then I would suggest to add a comment just above to explain it.